src/server/src/main.rs

```diff
@@ -1,23 +1,248 @@
+use bastion::prelude::*;
+use ntex::http::StatusCode;
+use ntex::web::{App, HttpRequest, HttpResponse, HttpServer, Responder};
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex};
 use ntex::web;
+use serde::{Deserialize, Serialize};
 
-#[web::get("/")]
-async fn index() -> impl web::Responder {
-    "Hello, World!"
+// user
+#[derive(Deserialize, Serialize, Clone)]
+struct User
+{
+    username: String,
+    password: String,
+    email: Option<String>,
+    age: Option<u32>,
+    gender: Option<String>,
 }
 
-#[web::get("/{name}")]
-async fn hello(name: web::types::Path<String>) -> impl web::Responder {
-    web::HttpResponse::Ok()
-        .content_type("text/plain")
-        //THIS IS FOR THE DEMO, DO NOT DO THIS IN PRODUCTION GET RID OF THIS SOON
-        .set_header(ntex::http::header::ACCESS_CONTROL_ALLOW_ORIGIN, "*")
-        .body(format!("Hello {}!", &name))
+// sessionId -> child process reference
+//type SessionMap = Arc<Mutex<HashMap<String, ChildrenRef>>>; //when bastion is implemented
+type SessionMap = Mutex<HashMap<String, String>>;
```

Comment:
I would prefer sessions IDs as numbers rather than strings for speed. uint64s should work.